lib/tutorials/en_US/cookies.md

```diff
@@ -1,59 +1,98 @@
-## Cookies
+# Cookies
 
 _This tutorial is compatible with hapi v17_
 
-When writing a web application, cookies are often used to keep state about a user between requests. With hapi, cookies are flexible, secure, and simple to use.
+- [Overview](#overview)
+- [Configuring the Server](#server)
+    - [server.state()](#server.state)
+    - [route.options.state()](#options.state)
+- [Setting a Cookie](#setting)
+    - [h.state()](#h.state)
+- [Overriding Options](#override)
+- [Getting a Cookie Value](#value)
+- [Clearing a Cookie](#clearing)
 
-## Configuring the server
 
-hapi has several configurable options when dealing with cookies. The defaults are probably good for most cases, but can be changed when needed.
+## <a name="overview" /> Overview
 
-To use a cookie, first you should configure it by calling [`server.state(name, [options])`](/api#-serverstatename-options) where `name` is the name of the cookie, and `options` is an object used to configure the cookie.
+When writing a web application, cookies are often used to keep state about a user between requests. With hapi, cookies are flexible, secure, and simple to use. To use cookies in hapi, you first configure the cookie by calling `server.state()`, then set the cookie via the response toolkit.  
+
+## <a name="server" /> Configuring the server
+
+hapi has several configurable options when dealing with cookies. The defaults are probably good for most cases, but can be changed when needed. To prepare a cookie, you first need to name it and configure a list of options.  You do this by calling `server.state(name, [options])`.
+
+### <a name="server.state" /> server.state()
+
+To use a cookie, you first need to configure it by calling [`server.state(name, [options])`](/api#-serverstatename-options) where `name` is the name of the cookie, and `options` is an object used to configure the cookie:
 
 ```javascript
 server.state('data', {
     ttl: null,
     isSecure: true,
     isHttpOnly: true,
     encoding: 'base64json',
-    clearInvalid: false, // remove invalid cookies
-    strictHeader: true // don't allow violations of RFC 6265
+    clearInvalid: true,
+    strictHeader: true
 });
 ```
+In the example above, we first name the cookie `data`. We then configure the options to customize the cookie.
```

Comment:
Jarrod, what do you think of adding a note that not all cookie options are required?

I like your descriptions below and the default values are typically just fine in most cases. Developers don’t have to define all options on each cookie 😃

Reply:
Good suggestion, I'll put a note in about it